### ISSUE TITLE
Add random collectible coins 

### ISSUE THREAD

**Issue by r4pt0s** [zero-to-mastery/ZTM-Quest#34]
To make the game more fun exploring, add coins randomly in each map. 
Feel free to search or create coin sprites which you can use.

The coins should be collectible by the player and the count should get stored in main characters props.

**Comment by Muhammad-Rebaal** [comment]
Please assign these to me

**Comment by r4pt0s** [comment]
Thanks @Muhammad-Rebaal .
Go for it 🚀🏃🏽‍♂️‍➡️

**Comment by r4pt0s** [comment]
@Muhammad-Rebaal are you still working on it? Because there came a pr #162 in which can resolve this issue.

**Comment by Muhammad-Rebaal** [comment]
Yes I'm working on it and gonna submit the pull request on Tuesday.

**Comment by r4pt0s** [comment]
Awesome @Muhammad-Rebaal.
Then I will wait 🙌🏻

**Comment by HASH-0021** [comment]
@r4pt0s Is there any update regarding this issue?

**Comment by r4pt0s** [comment]
Hi @HASH-0021 ,

@Muhammad-Rebaal was/is working on it. Therefore I didn't merge your PR till now.

I want to wait for @Muhammad-Rebaal because he was first on it. 🙏🏻

**Comment by Muhammad-Rebaal** [comment]
Apologize @r4pt0s !
Unfortunately there is an issue caused with my pc due to which I'm unable to continue .

**Comment by r4pt0s** [comment]
No problem @Muhammad-Rebaal .
Then I will merge @HASH-0021 PR.
🙌🏻
